examples/anvil/examples/local_anvil.rs

```diff
@@ -0,0 +1,15 @@
+//! Example of spinning up a local Anvil node.
+
+use alloy_node_bindings::Anvil;
+use eyre::Result;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Spin up a local Anvil node.
+    // Ensure `anvil` is available in $PATH
+    let anvil = Anvil::new().spawn();
```

Comment:
this should also use a few built in functions, like block time etc...
```suggestion
    let anvil = Anvil::new().try_spawn()?;
```